File: 01_ProgrammingBasics/_01_FirstStepsInProgrammingExercise/_02_RadiansToDegrees.java
package _01_ProgrammingBasics._01_FirstStepsInProgrammingExercise;

import java.util.Scanner;

public class _02_RadiansToDegrees {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input radians
        double radians = Double.parseDouble(scan.nextLine());

        // Convert radians to degrees
        double degrees = radians * 180 / Math.PI;

        // Print the converted degrees
        System.out.println(degrees);
        
        scan.close();
    }
}

File: 23_C#Basics/_01_FirstStepsInProgrammingExercise/_02_RadiansToDegrees/_02_RadiansToDegrees.cs
using System;

public class _02_RadiansToDegrees
{
    public static void Main(string[] args)
    {
        // Read input radians
        double radians = double.Parse(Console.ReadLine());

        // Convert radians to degrees
        double degrees = radians * 180 / Math.PI;

        // Print the converted degrees
        Console.WriteLine(degrees);
    }
}
